--- ccache-3.2.1.orig/configure	2014-12-11 03:40:54.000000000 +0800
+++ ccache-3.2.1/configure	2014-12-28 00:22:44.013926605 +0800
@@ -5592,9 +5592,9 @@
 fi
 
 if test x${with_bundled_zlib} = x; then
-    { $as_echo "$as_me:${as_lineno-$LINENO}: checking for zlib >= 1.2.3" >&5
-$as_echo_n "checking for zlib >= 1.2.3... " >&6; }
-if ${ccache_cv_zlib_1_2_3+:} false; then :
+    { $as_echo "$as_me:${as_lineno-$LINENO}: checking for zlib >= 1.2.1" >&5
+$as_echo_n "checking for zlib >= 1.2.1... " >&6; }
+if ${ccache_cv_zlib_1_2_1+:} false; then :
   $as_echo_n "(cached) " >&6
 else
   cat confdefs.h - <<_ACEOF >conftest.$ac_ext
@@ -5604,9 +5604,9 @@
 main ()
 {
 
-            #if (ZLIB_VERNUM >= 0x1230)
+            #if (ZLIB_VERNUM >= 0x1210)
             #else
-            #error "ZLIB_VERNUM < 0x1230"
+            #error "ZLIB_VERNUM < 0x1210"
             #endif
 
   ;
@@ -5614,14 +5614,14 @@
 }
 _ACEOF
 if ac_fn_c_try_compile "$LINENO"; then :
-  ccache_cv_zlib_1_2_3=yes
+  ccache_cv_zlib_1_2_1=yes
 else
-  ccache_cv_zlib_1_2_3=no
+  ccache_cv_zlib_1_2_1=no
 fi
 rm -f core conftest.err conftest.$ac_objext conftest.$ac_ext
 fi
-{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ccache_cv_zlib_1_2_3" >&5
-$as_echo "$ccache_cv_zlib_1_2_3" >&6; }
+{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $ccache_cv_zlib_1_2_1" >&5
+$as_echo "$ccache_cv_zlib_1_2_1" >&6; }
     { $as_echo "$as_me:${as_lineno-$LINENO}: checking for gzdopen in -lz" >&5
 $as_echo_n "checking for gzdopen in -lz... " >&6; }
 if ${ac_cv_lib_z_gzdopen+:} false; then :
@@ -5662,7 +5662,7 @@
   true
 fi
 
-    if test $ccache_cv_zlib_1_2_3 = yes && test $ac_cv_lib_z_gzdopen = yes; then
+    if test $ccache_cv_zlib_1_2_1 = yes && test $ac_cv_lib_z_gzdopen = yes; then
         use_bundled_zlib=no
     else
         { $as_echo "$as_me:${as_lineno-$LINENO}: WARNING: using bundled zlib" >&5
